qgreenland/util/config.py

```diff
@@ -171,6 +174,8 @@ def export_config(cfg, output_path='./layers.csv'):
         'Data Source Abstract': layer['dataset']['metadata']['abstract'],
         'Data Source Citation': layer['dataset']['metadata']['citation']['text'],
         'Data Source Citation URL': layer['dataset']['metadata']['citation']['url'],
+        'Layer Size': (naturalsize(Path(get_layer_path(layer)).stat().st_size)
+                       if layer['dataset']['access_method'] != 'gdal_remote' else ''),
```

Comment:
Should we use `naturalize` to give us a human-readable size, or report in bytes so that e.g., summing the size of all RACMO layers is possible?